server/server.js

```diff
@@ -0,0 +1,41 @@
+/* eslint no-console:0 */
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+var ServiceManager = require('./service-manager');
+
+function server(serviceManager) {
```

Comment:
@sam-github this is more of a factory (like `loopback()` and `express()`). Notice the lack of `this` usage in the body.